README.md

```diff
@@ -11,6 +11,10 @@ The Temporal CLI is a command-line interface for running Temporal Server and int
 
 ### Installation
 
+#### curl
+
+`curl -sSf https://temporal.download/cli/ | sh`
```

Comment:
Personally, I would prefer `https://temporal.download/cli/install.sh`

Reply:
same :)

Reply:
as discussed internally updated to https://temporal.download/cli.sh